pkg/apis/routing/v1alpha1/interfaces.go

```diff
@@ -42,9 +42,9 @@ type Router interface {
 	GetStatusManager() *RouterStatusManager
 	// GetEventTypes returns the event types generated by the router.
 	GetEventTypes() []string
-	// AsRouter returns a unique reference to the router suitable for
-	// use as a CloudEvent router attribute.
-	AsRouter() string
+	// AsEventSource returns a unique reference to the router suitable for
+	// use as a CloudEvent source attribute.
+	AsEventSource() string
```

Comment:
This was probably renamed as part of a mass string replacement, but the meaning of the method is really `AsEventSource`. As in "return (self) as a CloudEvent 'source' attribute".